CHANGELOG.md

```diff
@@ -12,6 +12,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - added new method `Grid::delete_orders` and the corresponding switch
   `--delete-orders` in the subcommand `write` of the CLI
 
+### Changed
+
+- renamed `pineappl.lumi.rs` to `pineappl.channel.rs`
```

Comment:
This should probably talk about renaming the modules. What's unclear is that the first `.` refers to a module and the second one to file suffix.

Reply:
I adjusted my suggested change to reflect that - do you agree with it?